### PR TITLE
added details for object store usage SF 9x

### PR DESCRIPTION
**Pull request by luizamujdei**

### CODE REVIEW

File: modules/ROOT/pages/salesforce/salesforce-connector-95.adoc [hunk 1]
```diff
@@ -801,11 +801,60 @@ The job detail page in Salesforce includes a related list of all the batches for
...
+When configuring an OAuth 2.0 connection, you can specify an object store that stores each resource owner's ID data. If you don't specify an object store, Mule automatically provisions the default object store.
+
+The app interacts with the object store automatically when a new resource owner is authenticated, the access token is refreshed, or the access token is invalidated.
+
+=== Message Redelivery for Input Sources
+
+You can configure a redelivery policy for input sources by setting the number of redelivery attempts to try after an initial failure. You can specify an object store for this policy. If you don't specify an object store, Mule creates a non-persistent object store.
+
+Based on the number of retries configured for the
+redelivery policy, The number of transactions used to interact with the object store varies based on the number of retries configured for the redelivery policy.
```
Comment (alexandruatomei9) [inline modules/ROOT/pages/salesforce/salesforce-connector-95.adoc:856]: don't think there's a need for line break here

File: modules/ROOT/pages/salesforce/salesforce-connector-95.adoc [hunk 2]
```diff
@@ -801,11 +801,60 @@ The job detail page in Salesforce includes a related list of all the batches for
...
+. Click YOUR_NAME > *Setup* > *Monitoring* > *Bulk Data Load Jobs*.
 . Click the *Job ID* to view the job detail page.
 
 The job detail page includes a related list of all the batches for the job. The related list provides View Request and View Response links for each batch. If the batch is a CSV file, the links return the request or response in CSV format. If the batch is an XML file, the links return the request or response in XML format. These links are available for batches created in Salesforce API version 19.0 and later.
 
+== Object Store Usage
+
+Both Salesforce Connector and Mule use an object store to persist data for features such as automatic message replay and message redelivery:
+
+* A Mule app that runs on-premises uses Mule Object Store which has no transaction limits.
```
Comment (kbostrom) [inline modules/ROOT/pages/salesforce/salesforce-connector-95.adoc]:
```suggestion
* A Mule app that runs on-premises uses Mule Object Store, which has no transaction limits.
```

File: modules/ROOT/pages/salesforce/salesforce-connector-95.adoc [hunk 3]
```diff
@@ -801,11 +801,60 @@ The job detail page in Salesforce includes a related list of all the batches for
...
+
+* A Mule app with a CloudHub deployment uses Object Store v2. The free version of Object Store v2 has a limit of 10 transactions per second.
```
Comment (kbostrom) [inline modules/ROOT/pages/salesforce/salesforce-connector-95.adoc]:
```suggestion
* A Mule app with a CloudHub deployment uses Object Store v2. 
The free version of Object Store v2 has a limit of 10 transactions per second.
```

**Review by kbostrom — APPROVED** [review]
Done

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**